app/javascript/components/pages/ProtectedReviewIndexPage.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+import { Card, CardBody, CardTitle, CardText, Button, Row  } from 'reactstrap';
+import { Nav, NavItem } from 'reactstrap'
+
+class ProtectedReviewIndexPage extends Component {
+    render() {
+
+            return(
+                <>
+                <h1>My Reviews</h1>
+                {this.props.review && this.props.schools &&
+                    <div>
+                        <Row sm="3">
+                            {this.props.review && this.props.review.map(review => {
+                                return(
+                                    <Card key={review.id}>
+                                            <CardBody>
+                                                <CardTitle>{review.title}</CardTitle>
+                                                <CardText>{review.review_text}</CardText>
+                                                <CardText>User ID: {review.user_id}</CardText>
+                                                <CardText>School ID: {review.school_id}</CardText>
```

Comment:
You usually won't want to display the user id, and unless the user needs the school id for a particular reason, it is not necessary to be on the card.